File: carrito_v4/src-gen/beans/Provider.java
//Start of user code imports
import java.io.Serializable;
//End of user code

public class Provider  implements Serializable{
	

	/**
	* the ruc attribute.
	*/
	private String ruc;


	/**
	* the idProvider attribute.
	*/
	private String idProvider;


	/**
	* the nombre attribute.
	*/
	private String nombre;


	/**
	* the description attribute.
	*/
	private String description;
	

	/**
	* Empty Constructor.
	*/
	public Provider (){
	}

	/**
	* Constructor.
	*/
	public Provider (String ruc, String idProvider, String nombre, String description){
		this.ruc = ruc;
		this.idProvider = idProvider;
		this.nombre = nombre;
		this.description = description;
	}
 
	public String getRuc() {
       return this.ruc;
     }

	public String getIdProvider() {
       return this.idProvider;
     }

	public String getNombre() {
       return this.nombre;
     }

	public String getDescription() {
       return this.description;
     }
 
     public void setRuc(String ruc) {
        this.ruc = ruc;
     }

     public void setIdProvider(String idProvider) {
        this.idProvider = idProvider;
     }

     public void setNombre(String nombre) {
        this.nombre = nombre;
     }

     public void setDescription(String description) {
        this.description = description;
     }


}

File: carrito_v4/src-gen/windows/Model/Provider.cs
//Start of user code imports
using System;
using System.Runtime.Serialization;
using System.Collections.Generic;
//End of user code

namespace CarritoDeCompras.Model
{
    [DataContract]
    public class Provider 
    {
		//The Id property is marked as the Primary Key
        [SQLite.PrimaryKey, SQLite.AutoIncrement]
        [DataMember]
        public int Id { get; set; }

		[DataMember]
		public string description { get; set; }

		[DataMember]
		public int idProvider { get; set; }

		[DataMember]
		public string ruc { get; set; }

		[DataMember]
		public string nombre { get; set; }

		/**
		* Empty Constructor.
		*/
		public Provider (){
		}
	
		/**
		* Constructor.
		*/
		public Provider (string description, int idProvider, string ruc, string nombre){
			this.description = description;     
			this.idProvider = idProvider;     
			this.ruc = ruc;     
			this.nombre = nombre;     
		}

    }
}
